concourse/pipelines/windows-image-build.jsonnet

```diff
@@ -456,8 +456,8 @@ local imgpublishjob = {
       passed: [
         job.passed,
       ],
-      // Builds are automatically pushed to testing. Triggering staging will automatically progress to prod and internal.
-      trigger: if job.env == 'staging' then false else true,
+      // Builds are automatically pushed to testing.
```

Comment:
We need it to stop after testing. Staging was useful because it was the common step between testing and "the rest" internal/internal, prod etc.

I'm not sure what the right condition for this is, but I'll think on it tonight.

Reply:
Yeah.... I was thinking about that- internal maybe?

Reply:
sql_envs and windows_install_media_envs don't use the internal project. But maybe best approach is to create internal project(s) for them so it's a predictable pattern.

Reply:
How about this.